src/cordial_sound/scripts/sound_listener.py

```diff
@@ -43,7 +41,7 @@ def play_sound(data):
             output_device_index=SPEAKER_DEVICE_INDEX,
         )
         stream.write(data.data)
-        time.sleep(1)
+        time.sleep(0.05)
```

Comment:
It would be nice to also make this a parameter. A good general rule is that whatever you're hardcoding should be exposed in some way - in ROS that typically means through parameters.